scripts/scenarios/cloud9/qa/no-ssl/qa-scenario-8a.yaml

```diff
@@ -172,26 +172,10 @@ proposals:
       heat-server:
       - cluster:services
 
-- barclamp: ceilometer
-  attributes:
-  deployment:
-    elements:
-      ceilometer-agent:
-      - "@@compute-kvm@@"
-      - "@@monasca-server@@"
-      ceilometer-agent-hyperv: []
-      ceilometer-central:
-      - cluster:services
-      ceilometer-server:
-      - cluster:services
-      ceilometer-swift-proxy-middleware: []
-
 - barclamp: monasca
   attributes:
   deployment:
     elements:
-      monasca-master:
```

Comment:
Why is this being removed?

Reply:
There is no longer a `monasca-master` role.